packages/lang/locales/en.json

```diff
@@ -268,7 +268,7 @@
   "purchasedProducts.outOfStock": "Out of stock",
   "purchasedProducts.minQuantity": "Min Quantity {minQuantity}",
   "purchasedProducts.maxQuantity": "Max Quantity {maxQuantity}",
-  "purchasedProducts.orderDialog.aditionalInformation": "Additional Information",
+  "purchasedProducts.orderDialog.additionalInformation": "Additional Information",
```

Comment:
FYI, we need to update these variables in the translation service csv

Reply:
thanks, didn't know. I think these are the only typos on this file, going forward cspell should prevent these.